contrib/pkg/utils/azure/azure.go

```diff
@@ -0,0 +1,26 @@
+package azure
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/openshift/hive/pkg/constants"
+)
+
+// GetCreds reads Azure credentials used for install/uninstall from either the default
+// credentials file (~/.azure/osServiceAccount.json), the standard environment variable,
+// or or provided credsFile location (in increasing order of preference).
+func GetCreds(credsFile string) ([]byte, error) {
+	credsFilePath := filepath.Join(os.Getenv("HOME"), ".azure", constants.AzureCredentialsName)
```

Comment:
Not all OS's expose a HOME variable...  there is a utility:
import `k8s.io/client-go/util/homedir` 
`homedir.HomeDir()`

Reply:
nice!